servers/rendering_server.cpp

```diff
@@ -1697,7 +1697,7 @@ void RenderingServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("texture_get_path", "texture"), &RenderingServer::texture_get_path);
 
 	ClassDB::bind_method(D_METHOD("texture_set_force_redraw_if_visible", "texture", "enable"), &RenderingServer::texture_set_force_redraw_if_visible);
-	ClassDB::bind_method(D_METHOD("texture_get_rd_texture", "texture", "srgb"), &RenderingServer::texture_get_rd_texture_rid, DEFVAL(false));
+	ClassDB::bind_method(D_METHOD("texture_get_rd_texture", "texture", "srgb"), &RenderingServer::texture_get_rd_texture);
```

Comment:
Did you mean to remove the default value for `srgb`? This breaks compat.

Reply:
@akien-mga AH! I get my mistake here, It's counting from the last parameter, I thought it was a bug and it was setting a default for texture :)

I shall fix!